daml-foundations/daml-ghc/daml-stdlib-src/DA/Next/Map.daml

```diff
@@ -121,6 +132,16 @@ insert k v (Map t) = Map $ TextMap.insert (keyToText k) v t
 union : MapKey k => Map k v -> Map k v -> Map k v
 union (Map t1) (Map t2) = Map $ TextMap.union t1 t2
 
+merge
```

Comment:
Some documentation for this function would be nice (I know what it does but our users might not)

Reply:
Oops, that got lost during copy'n'paste.